xlators/features/index/src/index.c

```diff
@@ -1961,7 +1982,7 @@ index_getxattr(call_frame_t *frame, xlator_t *this, loc_t *loc,
     return 0;
 }
 
-int64_t
+static int64_t
 index_fetch_link_count(xlator_t *this, index_xattrop_type_t type)
```

Comment:
```suggestion
index_fetch_link_count(char *index_basepath, index_xattrop_type_t type)
```

nit: you can pass 'priv->index_basepath' instead of 'this'. (Unrelated specifically here, but if you are already making the function static...)

Reply:
This is a backport, won't be making these changes.

Reply:
> This is a backport, won't be making these changes.

Ah, the title did not include this piece of information. Perhaps we should add a version or just 'Backport: ...' for such efforts.

Reply:
We used to follow that in gerrit, now we just add a label on the PR saying release-9/release-8 etc.